### PR TITLE
[MOBILE-1279] Add data collection enabled and push token registration enabled

### PR DESCRIPTION
**Pull request by Ulrico972**
### What do these changes do?
Add data collection enabled and push token registration enabled

### Why are these changes necessary?
To keep cordova module up to date with the Airship native library features

### How did you verify these changes?
By adding settings in the sample and manually testing the different cases

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -192,7 +192,7 @@
                 <source url="https://github.com/CocoaPods/Specs.git"/>
             </config>
             <pods use-frameworks="true">
-                <pod name="Airship" spec="13.0.4" />
+                <pod name="Airship" spec="13.1.0" />
```
Comment (BrianBatchelder) [inline plugin.xml]: You need to rebase on master. This change has already been made. There might be others.

File: www/UrbanAirship.js [hunk 2]
```diff
@@ -965,5 +965,25 @@ module.exports = {
     }
 
     _runAction("add_custom_event_action", actionArg, success, failure)
+  },
+
```
Comment (BrianBatchelder) [inline www/UrbanAirship.js]: Can you add a doc block to each of these? See previous functions for examples.

File: plugin.xml (hunk 1 above)
Comment (Ulrico972) [reply plugin.xml]: Done

File: www/UrbanAirship.js (hunk 2 above)
Comment (Ulrico972) [reply www/UrbanAirship.js]: Done

File: www/UrbanAirship.js [hunk 3]
```diff
@@ -980,21 +980,53 @@ module.exports = {
     callNative(success, failure, "trackScreen", [screen])
   },
 
+  /**
+   * Enables or disables data collection.
+   *
+   * @param {boolean} enabled true to enable data collection, false to disable.
```
Comment (BrianBatchelder) [inline www/UrbanAirship.js]:
```suggestion
   * @param {boolean} set true to enable data collection, false to disable.
```

File: www/UrbanAirship.js [hunk 4]
```diff
@@ -980,21 +980,53 @@ module.exports = {
...
+   */
   isDataCollectionEnabled: function(success, failure) {
     argscheck.checkArgs('fF', 'UAirship.isDataCollectionEnabled', arguments)
     callNative(success, failure, "isDataCollectionEnabled")
   },
 
+  /**
+   * Enables or disables push token registration.
+   *
+   * @param {boolean} enabled true to enable push token registration, false to disable.
```
Comment (BrianBatchelder) [inline www/UrbanAirship.js]:
```suggestion
   * @param {boolean} set true to enable push token registration, false to disable.
```

File: www/UrbanAirship.js (hunk 3 above)
Comment (Ulrico972) [reply www/UrbanAirship.js]: I'm not sure we should change it, enabled is just the variable name =)

File: www/UrbanAirship.js (hunk 4 above)
Comment (Ulrico972) [reply www/UrbanAirship.js]: Same here

Comment (BrianBatchelder) [reply www/UrbanAirship.js]: OK

File: www/UrbanAirship.js (hunk 3 above)
Comment (BrianBatchelder) [reply www/UrbanAirship.js]: OK

**Comment by BrianBatchelder** [comment]
@Ulrico972 I merged this PR so I could move on to the release.

**Pull request merged**